### PR TITLE
feat(alicefate): add "deleted" to ideaCard.status

### PR DESCRIPTION
**Pull request by alicefate**

### CODE REVIEW

File: api/idea-card/models/idea-card.settings.json [hunk 1]
```diff
@@ -75,7 +75,8 @@
         "applying",
         "approved",
         "archived",
-        "workshopping"
+        "workshopping",
+        "deleted"
```
[the remark this answers is not in the document]
Comment (alicefate) [reply api/idea-card/models/idea-card.settings.json:79]: @chungthuang user can't access to deleted idea, can we use updated_by as deleted_at?

[the remark this answers is not in the document]
Comment (chungthuang) [reply api/idea-card/models/idea-card.settings.json:79]: How would you tell it's a delete using updated_by? With a `deleted_at` field, the data is still in database, but the frontend can display it as deleted.

[the remark this answers is not in the document]
Comment (alicefate) [reply api/idea-card/models/idea-card.settings.json:79]: I will suggest use custom controllers in strapi, filter out deleted ideaCard. Once user delete an ideaCard, it should never return to frontend.

**Comment by becevka** [comment]
This was already added in Strapiv4 repo, closing for clean up

**Pull request closed without merging**